plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/src/com/google/cloud/tools/eclipse/appengine/libraries/ui/ApiClientLibrariesPage.java

```diff
@@ -16,12 +16,34 @@
 
 package com.google.cloud.tools.eclipse.appengine.libraries.ui;
 
+import com.google.cloud.tools.eclipse.appengine.facets.AppEngineStandardFacet;
 import com.google.cloud.tools.eclipse.appengine.libraries.model.CloudLibraries;
+import com.google.cloud.tools.eclipse.appengine.ui.AppEngineImages;
+import com.google.common.collect.Maps;
+import java.util.Map;
+import org.eclipse.jdt.core.IClasspathEntry;
+import org.eclipse.jdt.core.IJavaProject;
 
 public class ApiClientLibrariesPage extends CloudLibrariesPage {
```

Comment:
So now `ApiClientLibrariesPage` is the only subclass of `CloudLibrariesPage`. The latter was to reuse common base, so I think we can just merge the former into the latter.